src/panels/developer-tools/statistics/dialog-statistics-fix-units-changed.ts

```diff
@@ -47,11 +46,13 @@ export class DialogStatisticsFixUnitsChanged extends LitElement {
         )}
       >
         <p>
-          The unit of this entity changed, we can't store values in multiple
-          units. <br />If the historic statistic values have a wrong unit, you
-          can update the units of the old values. The values will not be
-          updated.<br />Otherwise you can choose to delete all historic
-          statistic values, and start over.
+          The unit of this entity changed to
```

Comment:
`data.supported_unit` tells which unit is allowed, should we include that too or instead of `data.metadata_unit`?

Reply:
meaning a user can change their state to that unit to fix the issue?